core/src/foundations/README.md

```diff
@@ -51,39 +40,27 @@ Example:
 }
 ```
 
-The `tokens.js` script, responsible for generating CSS and SCSS variables along with utility classes, will follow the same nomenclature: `prefix - CSS property name - type variation - modifier name`.
-
-Examples:
+The `index.js` script, on the scripts/tokens folder, is responsible for generating SCSS variables and utility classes, will follow the same nomenclature: `prefix - CSS property name - type variation - modifier name`.
 
-CSS Custom Property:
-
-```css
---ionic-color-primary-100
-```
+Examples: 
 
 SCSS Variable:
 
 ```scss
-$ionic-color-primary-100
+$ion-semantics-primary-300
 ```
 
 Utility class:
 
 ```css
-.ionic-color-primary-100
+.ion-semantics-primary-300
 ```
 
-## When to change the Design Tokens
-
-It's not expected that the `design-tokens.json` needs to be changed frequently. This represents the base of all the Ionic Design System and any change to the json file should be taken carefully, synced with the UX/UI Team and reviewed by other devs.
-
-Adding new tokens is more or less risk free. However, changing values will result in visual breaking changes and renaming tokens might cause the build to be broken.
-
-## How to change the Design Tokens
+## Managing the Design Tokens
 
-Using `npm run build.tokens` will generate the `ionic.root.scss`, `ionic.vars.scss`, and `ionic.utility.scss` files with the updated values that come from the `design-tokens.json`.
+It's not expected that the Tokens need to be changed frequently. This represents the base of all the Ionic Design System and any change to them should be taken carefully, synced with the UX/UI Team and reviewed by other devs. 
 
-Design Tokens will also be generated automatically with the `npm run build` command. This ensures that if someone modifies the JSON without running the build tokens command, the changes will be applied correctly.
+With the current architecture, this management is done on the [outsystems-design-tokens repo](https://github.com/OutSystems/outsystems-design-tokens) and updated its dependency here when necessary.
```

Comment:
```suggestion
With the current architecture, this management is done on the [outsystems-design-tokens repo](https://github.com/OutSystems/outsystems-design-tokens), and its dependency is updated here as needed.
```

The last part is a bit confusing to me, so here's a suggestion on how to change the verbiage.